agot-bg-game-server/src/client/GameSettingsComponent.tsx

```diff
@@ -56,7 +57,16 @@ export default class GameSettingsComponent extends Component<GameSettingsCompone
                 <FormCheck
                     id="pbem-setting"
                     type="checkbox"
-                    label="PBEM"
+                    label={
+                        <OverlayTrigger overlay={
+                            <Tooltip id="pbem-tooltip">
+                                <b>P</b>lay <b>B</b>y <b>E</b>-<b>M</b>ail<br/>
+                                Players receive an e-mail when it is their turn.
+                                Those games are typically played over days or weeks.
+                            </Tooltip>}
+                            delay={{show: 250, hide: 100}}>
```

Comment:
Unless you have a specific reasoning for keeping them, I think you can remove the delay here, and keep the default values.